packages/core/src/index.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { debug, expandVars, HoprDB, PublicKey } from '@hoprnet/hopr-utils'
+import HoprCoreEthereum from '@hoprnet/hopr-core-ethereum'
+import { expect } from 'chai'
+import PeerId from 'peer-id'
+import sinon from 'sinon'
+import Hopr, { HoprOptions, resolveEnvironment, VERSION } from '.'
+
+const log = debug('hopr-core:test:index')
+
+describe('hopr controller ', async function () {
```

Comment:
Please document tests better. Also what is a hopr controller? Never heard this before?

Reply:
Stealing the term `controller` from the good ol’ MVC (model-view-controller) architecture. Considering this is a wrapper class for the actual model (`hopr` class), the controller is the one that interacts with the underlying API.

Happy to rename it.